src/components/form/widgets/select.ts

```diff
@@ -73,3 +76,16 @@ function findValue(value: string, options: Option[]) {
 function findValues(value: string[], options: Option[]) {
     return options.filter((option: Option) => value.includes(option.value));
 }
+
+function getAdditionalProps(schema: any) {
+    let props: any = {};
+
+    if (schema.lime?.component?.props) {
+        props = {
+            ...props,
```

Comment:
Super minor, but this seems a bit redundant since `props` is always an empty object here :sweat_smile:

Reply:
# 🤦‍♂️ 😂

I copied this from the input-field widget, and removed some code between the `let props` and the `if`, and obviously forgot to take a step back and look at the result.

Thanks for the catch!

Reply:
Oh… hang on… actually, I'm gonna have a look at if I can write a regression test for this, before we merge 😊

Reply:
Or… maybe I'm not… because there are no tests at all currently, and I don't really have time to write them from scratch right now… 😞

Reply:
Anyway, this is ⚡